Gemfile

```diff
@@ -158,4 +158,7 @@ gem 'httparty'
 # for setting middleware redirects
 gem 'rack-host-redirect'
 
+# adding simple jquery news ticker -- integrates https://github.com/BenjaminRH/jquery-ticker
+gem 'ticker-rails'
```

Comment:
I'll have to wait until I switch over to pull the branch locally, but I don't understand why we need a gem for this.

Reply:
we don't need this gem. for some reason it stayed from when i was working on another branch.  I'll remove it from the gemfile and re-push